.github/workflows/typos.yml

```diff
@@ -0,0 +1,21 @@
+name: typos
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Change ˙ubuntu-latest˙ to ˙windows-latest˙, since my project doesnt compile to linux as of now

Reply:
This is just a spell checker,not a compilation process.So choosing ubuntu will run this action faster than choosing windows.